src/Expr.js

```diff
@@ -263,6 +249,70 @@ var exprToString = function(expr, caller) {
     )
   }
 
+  if ('collections' in expr) {
+    if (expr['collections'] === null) {
+      return 'Collections()'
+    }
+  }
+
+  if ('databases' in expr) {
+    if (expr['databases'] === null) {
+      return 'Databases()'
+    }
+  }
+
+  if ('select' in expr) {
+    return (
+      'Select(' +
+      exprToString(expr['select']) +
+      ', ' +
+      exprToString(expr['from']) +
+      ')'
+    )
+  }
+
+  if ('if' in expr) {
+    return (
+      'If(' +
+      exprToString(expr['if']) +
+      ', ' +
+      exprToString(expr['then']) +
+      ', ' +
+      exprToString(expr['else']) +
+      ')'
+    )
+  }
+
+  if ('contains_path' in expr) {
+    return (
+      'ContainsPath(' +
+      exprToString(expr['contains_path']) +
+      ', ' +
+      exprToString(expr['in']) +
+      ')'
+    )
+  }
+
+  if ('contains_field' in expr) {
+    return (
+      'ContainsField(' +
+      exprToString(expr['contains_field']) +
+      ', ' +
+      exprToString(expr['in']) +
+      ')'
+    )
+  }
+
+  if ('contains_value' in expr) {
+    return (
+      'ContainsValue(' +
+      exprToString(expr['contains_value']) +
+      ', ' +
+      exprToString(expr['in']) +
+      ')'
+    )
+  }
+
```

Comment:
no need to add those special cases for those function it is already sneak cased, and the driver logic will convert it to camel case

Reply:
@marrony I had originally removed these, however, when I run my unit tests locally with the `nightly:latest` image, these are failing 🤔 Any idea why that might be?